pkg/asset/manifests/azure/cloudproviderconfig.go

```diff
@@ -56,6 +57,11 @@ func (params CloudProviderConfig) JSON() (string, error) {
 		// https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-tcp-reset
 		LoadBalancerSku: "standard",
 	}
+
+	if params.ARO {
+		config.authConfig.UseManagedIdentityExtension = false
```

Comment:
Use a `useManagedIdentity` variable to populate the `UseManagedIdentityExtension` field.